readme.md

```diff
@@ -27,8 +27,8 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://Svimpik.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://Svimpik.github.io/layout_moyo-header/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="931" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/cd803f3c-46ad-4fed-ae64-c1bc875b6ce8">